spec/helpers/loss_report_helper_spec.rb

```diff
@@ -0,0 +1,54 @@
+
+describe 'Loss Report Helper' do
+  let(:umm_c_version) { '1.15.3' }
+
+  context '#prepare_collections' do
```

Comment:
this is no longer in the helper file so shouldn't be here, but that doesn't mean you can't test it in a controller test